demos/qadi_experiments/qadi_google_real.py

```
@@ -0,0 +1,141 @@
+                key, value = line.strip().split('=', 1)
+                os.environ[key] = value.strip('"').strip("'")
+
+sys.path.insert(0, str(Path(__file__).parent / 'src'))
+
+async def run_minimal_qadi(prompt: str):
+    """Run minimal QADI cycle optimized for Google API."""
+    from mad_spark_alt.core.llm_provider import setup_llm_providers, llm_manager, LLMRequest
+    from mad_spark_alt.agents.questioning.llm_agent import LLMQuestioningAgent
+    from mad_spark_alt.agents.abduction.llm_agent import LLMAbductiveAgent
+    from mad_spark_alt.core.answer_extractor import TemplateAnswerExtractor
+    from mad_spark_alt.core.interfaces import IdeaGenerationRequest, ThinkingMethod
```

Ruff (0.11.9)

`mad_spark_alt.core.interfaces.ThinkingMethod` imported but unused

Remove unused import: `mad_spark_alt.core.interfaces.ThinkingMethod`

(F401)